2.4/root/usr/share/container-scripts/mongodb/common.sh

```diff
@@ -244,7 +244,16 @@ function setup_keyfile() {
     exit 0
```

Comment:
Yes, but others are `exit 1` that's signalized about fatal errors while this case isn't an error.